lib/pacemaker/pcmk_output.c

```diff
@@ -728,9 +728,10 @@ pacemakerd_health_xml(pcmk__output_t *out, va_list args)
         state_s = pcmk_pacemakerd_api_daemon_state_enum2text(state);
     }
 
-    pcmk__output_create_xml_node(out, pcmk__s(sys_from, "<null>"),
+    pcmk__output_create_xml_node(out, "pacemakerd-health",
+                                 "sys_from", sys_from,
                                  "state", state_s,
-                                 "last_updated", pcmk__s(last_updated, ""),
+                                 "last_updated", last_updated,
```

Comment:
Changing the element name isn't really backward compatible since existing scripts won't know that an alternative is possible. Let's keep it as pacemakerd, always hard-coded (I'm not sure why it was sys_from, but that wasn't a good idea -- the whole point of the schema is to have something that scripts can rely on).

Reply:
I was concerned about that. Sounds good, I'll change it to `pacemakerd`.